ci-operator/config/openstack-k8s-operators/designate-operator/OWNERS

```diff
@@ -0,0 +1,23 @@
+approvers:
+  - abays
+  - beagles
+  - johnsom
+  - dprince
+  - olliewalsh
+  - Sandeepyadav93
+  - stuggi
+  - viroel
+  - weinimo
+  - gibizer
+
+reviewers:
+  - abays
+  - beagles
+  - johnsom
```

Comment:
johnsom is not part of OpenShift CI org, we need to ask him to join the org.

Reply:
Yes, we requested him to join the Org.